docs/components/Toast/Mobile.stories.tsx

```diff
@@ -20,11 +21,14 @@ export default {
     },
   },
   decorators: [
-    Story => (
-      <div style={{ height: "90vh" }}>
-        <Story />
-      </div>
-    ),
+    Story => {
+      const height = window.innerHeight * 0.9;
```

Comment:
Did this because `vh` doesn't work with React Native. May need to update this when we get things running through expo

Reply:
This only works if you load that page directly. If you're browsing from another place to here, that height is the initial height of the iframe since it animates from really big to really small. 

I'd actually suppress the error here since decorators are internal to storybook and won't show on the code tab.